src/vs/workbench/contrib/testing/common/testTypes.ts

```diff
@@ -687,80 +687,17 @@ export abstract class AbstractIncrementalTestCollection<T extends IncrementalTes
 
 		for (const op of diff) {
 			switch (op.op) {
-				case TestDiffOpType.Add: {
-					const internalTest = InternalTestItem.deserialize(op.item);
-					const parentId = TestId.parentId(internalTest.item.extId)?.toString();
-					if (!parentId) {
-						const created = this.createItem(internalTest);
-						this.roots.add(created);
-						this.items.set(internalTest.item.extId, created);
-						changes.add(created);
-					} else if (this.items.has(parentId)) {
-						const parent = this.items.get(parentId)!;
-						parent.children.add(internalTest.item.extId);
-						const created = this.createItem(internalTest, parent);
-						this.items.set(internalTest.item.extId, created);
-						changes.add(created);
-					}
-
-					if (internalTest.expand === TestItemExpandState.BusyExpanding) {
-						this.busyControllerCount++;
-					}
+				case TestDiffOpType.Add:
```

Comment:
The logic in these cases were moved down verbatim. I was originally considering overriding them and didn't, but I think this is still some decent cleanup